rest-gateway/src/main/scala/za/co/absa/spline/gateway/rest/DocumentationConfig.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ package za.co.absa.spline.gateway.rest
+
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.{EnableWebMvc, ResourceHandlerRegistry, WebMvcConfigurer}
+
+
+@EnableWebMvc
+@Configuration
+class DocumentationConfig extends WebMvcConfigurer {
+
+
+  override def addResourceHandlers(registry: ResourceHandlerRegistry) = {
+    registry
+      .addResourceHandler("/**")
+      .addResourceLocations(
+        "classpath:/META-INF/resources/",
```

Comment:
Generally classpath resources suppose to contain files needed for the application to work correctly (e.g. config files, mappings, definitions etc), or another static resources if the final build artifact is JAR-file.
In WAR-files however, the content that is accessible from the web should normally be place to the web resources dedicated place (i.e. `src/webapp/*`). Among other reasons there is a security reason - you do not need to serve the content of a classpath or META-INF resources, that is considered sensitive.

Reply:
ok, removed the resourceLocation of META-INF (this is not needed anyway).